### DISCUSSION TITLE
how to enter hover documentation through shortcut keys such as <c-h/j/k/l>

### DISCUSSION THREAD

**Discussion by gitttyyy** [LazyVim/LazyVim#2820] [Q&A] [answered]
Now I only can move into it by mouse action,It`t a little inconvenient when I need to scroll down or do some translate action
![image](https://github.com/LazyVim/LazyVim/assets/154318356/e2acbc3c-7fb6-433c-9a12-5ac7caf54704)

**Comment by dpetka2001** [comment, marked as answer]
`<C-f>`/`<C-b>` will scroll you through the docs window. You can even press `K` one more time and your cursor will go inside the docs window.

**Reply by gitttyyy** [reply]
Thank you, that's exactly what I wanted
